commands2/trigger.py

```diff
@@ -0,0 +1,149 @@
+from typing import Callable, Optional, overload, List, Union
+
+from ._impl import Command, Subsystem
+from ._impl import Trigger as _Trigger
+
+from .coroutinecommand import CoroutineCommand, Coroutineable, Coroutine
+
+
+class Trigger:
```

Comment:
Missing the following API:

* WhileActiveContinous
* WhileActiveOnce
* ToggleWhenActive
* CancelWhenActive
* Debounce